README.md

```diff
@@ -13,21 +12,45 @@ Teams spend significant time building parts of the design system (colors, variab
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
 
+:star: We appreciate your star, it helps!
+
+## Key features & benefits
+### Unopionated
+These days, there are a million and one great ways to build your next Web site or application. So, we have made being unopinionated the center of every decision made in FAST-DNA. 
+
+This principle of being unopinionated manifests in several important ways including:
+* A flat component architecture that lets you compose what you need without struggling with rigid patterns and complex objects.
+* Separating base components and style that supports multiple frameworks without re-writing or duplicating styles. This separation also lets you build your own styles, with your style technology of choice, without having to re-build or duplicate the base components.
+* Framework agnostic tooling that lets you use our Development tools with any view framework.
+* The ability to replace almost any FAST-DNA package with your package of choice. Get started with our animation or layout package and add more as you need them. Alternatively, use our suite of packages to build your next project from the ground up; it's your call.
+
+### UI development and style guide tools
+When developing components and views, excellent development tooling can make all the difference. FAST-DNA offers development and style guide tools that are with FAST components, components from other frameworks or your components. 
```

Comment:
I think the word "compatible" is likely missing here somewhere.

Reply:
fixed.